Famcy/_util_/_fpage.py

```diff
@@ -125,47 +128,76 @@ def register(cls, route, style, permission_level=0, background_thread=False, bac
 			else:
 				Famcy.FManager["MainBlueprint"].route(cls.route+"/bgloop")(bg_func)
 
-		if cls.event_source_flag:
-			Famcy.app.register_blueprint(Famcy.sse, url_prefix=cls.route+"/event_source")
-		
 	@classmethod
 	def render(cls, init_cls=None, *args, **kwargs):
+
+		# handle race condition issue: lock the function
+		#cls._lock.acquire()
+
+		# try:
 		route_list = request.path[1:].split("/")
 		route_name = '_'.join(route_list)
 
-		if g.sijax.is_sijax_request:
-			sijaxHandler = FSubmissionSijaxHandler
-			sijaxHandler.current_page = session.get(route_name+'current_page')
+		print("cls.style._sijax_enable: ", cls.style, cls.style._sijax_enable)
+		if cls.style._sijax_enable and request.method == 'POST' and g.sijax.is_sijax_request:
```

Comment:
The root cause for race condition -> separate sijax enable pages with normal pages